src/widget/wtimesignaturemenu.cpp

```diff
@@ -0,0 +1,82 @@
+#include "wtimesignaturemenu.h"
+
+#include <QtGui/QIntValidator>
+#include <QtWidgets/QHBoxLayout>
+
+namespace {
+constexpr int kMinBeatsPerBar = 2;
+constexpr int kMaxBeatsPerBar = 32;
+} // namespace
+
+WTimeSignatureMenu::WTimeSignatureMenu(QWidget* parent)
+        : QWidget(parent),
+          m_pBeatCountBox(make_parented<QSpinBox>(this)),
+          m_pBeatLengthBox(make_parented<QComboBox>(this)),
+          m_beat(mixxx::kInvalidFramePos) {
+    hide();
+    setWindowFlags(Qt::Popup);
+    setAttribute(Qt::WA_StyledBackground);
+    setObjectName("WTimeSignatureMenu");
+
+    QHBoxLayout* pMainLayout = new QHBoxLayout();
```

Comment:
I think the numbers should be arranged vertically as they are on sheet music.

Reply:
@Be-ing I was using the horizontal layout as inspired by musescore.
![Horizontal time signature boxes](https://musescore.org/sites/musescore.org/files/styles/width_740/public/TimeSig.jpg?itok=6nZcIBwR)

Reply:
I think adding a `/` in between should do the job.